source-install.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: all
+  vars:
+    ansible_python_interpreter: "/usr/bin/python3"
+    pulp_source_dir: "/home/vagrant/devel/pulp"
+    pulp_install_plugins:
+      pulp-python:
+        app_label: "pulp_python"
```

Comment:
We probably don't want pulp_python by default.

Reply:
+1, switching to pulp_file next push.

Reply:
It could be commented out, that was good in the other playbook file.